lib/src/cli/command/bump_command.dart

```diff
@@ -1,44 +1,47 @@
+import 'package:args/command_runner.dart';
 import 'package:cider/src/project.dart';
+import 'package:cider/src/cli/command/bump_sub_command.dart';
 import 'package:cider/src/cli/command/cider_command.dart';
 import 'package:version_manipulation/mutations.dart';
 
+enum BumpType {
+  breaking(mutation: BumpBreaking(), description: 'Bump the breaking version'),
+  major(mutation: BumpMajor(), description: 'Bump the major version'),
+  minor(mutation: BumpMinor(), description: 'Bump the minor version'),
+  patch(mutation: BumpPatch(), description: 'Bump the patch version'),
+  build(mutation: BumpBuild(), description: 'Bump the build version'),
+  pre(mutation: BumpPreRelease(), description: 'Bump the pre-release version'),
+  release(mutation: Release(), description: 'Bump the release version');
+
+  const BumpType({
+    required this.mutation,
```

Comment:
Is there a reason for named args? I think we're using regular positional arguments everywhere else.

Reply:
Ah, thanks, I didn't realize. It's a personal preference ;)